test/extensions/pt/text.yml

```diff
@@ -0,0 +1,262 @@
+name: "PT Text"
+
+features:
+  - namespaces
+  - experimentalFunctions
+
+documents:
+  - _id: "block-array-invalid-blocks-lenient"
+    _type: "test"
+    body: |
```

Comment:
Portable Text is typically not stored as a string, but as "real" arrays/objects.

Reply:
Fixed.